data/lib/compat/compat.lua

```diff
@@ -1006,3 +1006,13 @@ end
 function Guild.removeMember(self, player)
 	return player:getGuild() == self and player:setGuild(nil)
 end
+
+function doTileAddItemEx(pos, uid, flags)
+	flags = flags or 0
+	local tile = Tile(pos)
+	local item = Item(uid)
+	if tile and item then
+		tile:addItemEx(item, flags)
+	end
+	return false
```

Comment:
I need to check if item is not nil, calling the API just to check if something is waste.